decent-dao/fractal-interface
# Proposal menu item active on proposal details and create proposal pages

## Description

This PR implements marking sidebar navigation as active on underlying pages. That means, if there will be `treasury/some-stuff` page later on - sidebar navigation item will be marked as active without any changes in the codebase.

## Notes

This PR also contains minor `ProgressBar`

## Issue / Notion doc (if applicable)

Closes #873 

## Testing

1. Navigate to any DAO
2. Go to proposal details page - verify that Proposals sidebar menu item is active
3. Go to "Create Proposal" page - verify that Proposals sidebar menu item is active

## Screenshots (if applicable)
<img width="993" alt="Знімок екрана 2023-01-04 о 18 31 51" src="https://user-images.githubusercontent.com/25801162/210604529-cb156cc1-1a64-428e-9561-d0bd5b583196.png">
